docs/source/index.rst

```diff
@@ -1,7 +1,7 @@
 Augur Documentation
 ==================================
 
-Welcome to the Augur project's official documentation. Below is a brief primer on the project and its goals; if you're new to the project, we recommend giving it a once-over as a gentle introduction. Otherwise, feel free to hop right into the docs!
```

Comment:
The grammar is tricky, but it is actually correct how it is. 

https://www.quickanddirtytips.com/education/grammar/a-versus-an